### DISCUSSION TITLE
webhook image

### DISCUSSION THREAD

**Discussion by newfaboc** [kirimwa-id/forum#122] [Tanya Jawab] [answered]
Hi mas rio,

mau tanya untuk webhook incoming message berupa image dan caption apakah bisa didapatkan ?

Terima kasih.

**Comment by kafaa-dev** [comment, marked as answer]
Bisa mas, sesuai dokumentasi di https://developer.kirimwa.id/api/v1/#incoming-message-rules

![Screenshot_20220804-192713.jpg](https://user-images.githubusercontent.com/73120669/182847132-5fca0162-c71b-4c75-883b-14601b7b97f5.jpg)

**Reply by rioastamal** [reply]
@newfaboc Benar apa yang mas @syahid246 sampaikan, update webhook anda jika ingin memproses image. Nanti akan muncul `media_url`. Tapi ingat maksimum 100MB per bulan per akun, jika lebih dari itu maka nilai `media_url` akan menjadi `null`.
